tests/serac_thermal_solver.cpp

```diff
@@ -345,6 +345,47 @@ TEST(thermal_solver_rework, dyn_imp_solve_time_varying)
   MPI_Barrier(MPI_COMM_WORLD);
 }
 
+#ifdef MFEM_USE_AMGX
+TEST(thermal_solver, static_amgx_solve)
+{
+  MPI_Barrier(MPI_COMM_WORLD);
+
+  auto pmesh = buildBallMesh(10000);
+
+  auto params                                                   = ThermalConduction::defaultQuasistaticParameters();
+  std::get<IterativeSolverParameters>(params.T_lin_params).prec = AMGXPrec{.smoother = AMGXSolver::JACOBI_L1};
```

Comment:
Just setting the smoothing algorithm here was enough to have it converge in 1 iteration, vs the default block Jacobi smoother hitting the 500 iter cap and still being off by ~2 orders of magnitude over the tolerance

Reply:
Out of curiosity, what was the default smoother?

Reply:
The default is a block Jacobi method, I think. I've left that as the default in `solver_config.hpp`.